### ISSUE TITLE
[Epic] [Maintenance A11y Audit]: PCPG CH36

### ISSUE THREAD

**Issue by ohgreatkate** [department-of-veterans-affairs/va.gov-team#63362]
## Auditor(s)

@ohgreatkate @coforma-jamie @SarahKay8 

## Automated

- [x] Run axe checks on every page or unique state (required)
- [x] Run WAVE on every page if time allows. WAVE is more visual than axe or axe-coconut, but offers some excellent ways to identify nested headings and HTML5 landmark tags.

## Manual

- [X] Color contrast checks (recommend using whocanuse.com for beyond compliance reports)
- [X] Color blindness checks
- [X] Zoom layouts to 400% and inspect them for readability. If layouts break at 400%, I will start reducing them until they become stable, and log the zoom ratio when things started breaking.
- [X] Keyboard navigation

## Screen Reader

- [ ] Firefox and NVDA
- [x] Chrome and JAWS
- [X] iOS Safari and VoiceOver (Mobile)
- [X] Safari and Voiceover (Desktop)
- [ ] Chrome and Talkback

## Staging or Production Link

[staging link](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/introduction)

## Audit Spreadsheet
[Google sheets](https://docs.google.com/spreadsheets/d/1SCwKFlX57EMQ6DfP_ZkJ1YamNUtsaKbJBF2YhYh3H9E/edit?usp=sharing)

## Test Users

```bash
Use unauthenticated version:
1. Tab to the radio button and select "Veteran."
2. Tab and select "No."
3. Tab and select "No."
4. Tab and select "Yes."
5. Tab and select "Yes." This will give you a link to open the form.
6. Select "Start your application without signing in".
```

**Comment by ohgreatkate** [comment]
Audit findings have been delivered to product time via [Slack}(https://dsva.slack.com/archives/C057W3N9K2S/p1694550468904679?thread_ts=1693512707.420079&cid=C057W3N9K2S)
